sessionctx/binloginfo/binloginfo_test.go

```diff
@@ -483,3 +486,123 @@ func (s *testBinlogSuite) TestAddSpecialComment(c *C) {
 		c.Assert(re, Equals, ca.result)
 	}
 }
+
+func (s *testBinlogSuite) TestSequenceBinlog(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.Se.GetSessionVars().BinlogClient = s.client
+
+	tk.MustExec("drop sequence if exists seq")
+	// the default start = 1, increment = 1.
+	tk.MustExec("create sequence seq cache 3")
+	// trigger the sequence cache allocation.
+	tk.MustExec("select nextval(seq)")
+	sequenceTable := testGetTableByName(c, tk.Se, "test", "seq")
+	tc, ok := sequenceTable.(*tables.TableCommon)
+	c.Assert(ok, Equals, true)
+	_, end, round := tc.GetSequenceCommon().GetSequenceBaseEndRound()
+	c.Assert(end, Equals, int64(3))
+	c.Assert(round, Equals, int64(0))
+
+	// Check the sequence binlog.
+	// Got matched pre DDL and commit DDL.
+	var matched bool
+	for i := 0; i < 10; i++ {
```

Comment:
how about extract a function for line #510 ~ 521 and reuse in below.

Reply:
done